Snowflake.Data.Tests/HttpUtilTest.cs

```diff
@@ -36,5 +38,51 @@ public async Task TestIsRetryableHTTPCode(HttpStatusCode statusCode, bool forceR
 
             Assert.AreEqual(expectedIsRetryable, actualIsRetryable);
         }
+
+        [Test]
+        public void ShouldCreateHttpClientHandlerWithProxy()
+        {
+            // given
```

Comment:
is it a new practice to write comments give/when/then

Reply:
It is quite a new practice (previously I had added this convention in a few tests).
But even though previously it was not followed why not to introduce a good practice?